File: 125-valid-palindrome/valid-palindrome.cpp
class Solution 
{
public:
    bool isPalindrome(string s) 
    {
        string res="";
        for(char x:s)
        {
            if(x>='0' && x<='9') res+=x;
            if(x<='z' && x>='a') res+=x;
            if(x>='A' && x<='Z') res+=(x+('a'-'A'));
        }  
        int n = res.size();
        cout << res;
        for(int i=0;i<n/2;i++)
        {
            if(res[i]!=res[n-1-i]) return false;
        }
        return true;
    }
};

File: 125-valid-palindrome/valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        res = ""
        for x in range(len(s)):
            if s[x] >= 'a' and s[x] <= 'z':
                res += s[x]
            elif s[x] >= '0' and s[x] <= '9':
                res += s[x]
            elif s[x] >= 'A' and s[x] <= 'Z':
                res += chr(ord(s[x])+32)
        Y = res[::-1]
        return Y == res
        